build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.19.10-bookworm
+FROM golang:1.19.10-buster
```

Comment:
Should we add a comment explaining why this needs to be kept as `-buster`?

Reply:
Updated!

Reply:
🙌